src/cpp/dpm_sampler.cpp

```diff
@@ -558,7 +558,7 @@ void read_data(char* filename, std::ofstream& out_file)
     fclose(data);
 
     if (n == 1) {
-        out_file << "Nothing to do, have only one read... (n == 1)\n";
+        out_file << "Nothing to do, have only one read... (n == 1)" << std::endl;
```

Comment:
still and `endl` here

Reply:
Yes, this one I put on purpose to flush to output.

Reply:
Yes, this one I put on purpose to flush to output.

Reply:
alrighty then